Sources/TarkaUI/Components/Input Field/Model/TUIInputFieldState.swift

```diff
@@ -49,25 +49,38 @@ public enum TUIInputFieldState: Equatable {
   
   case none
   case focused
-  case inactive(String)
-  case error(String)
-  case alert(String)
-  case success(String)
-  case disabled(String)
+  case inactive(String, showIcon: Bool = true)
```

Comment:
Better create stored property and also add view modifier. Because it's not required for all cases

Reply:
But in the Figma i was able to see the components can have options to show/hide the icon